portal/models/i18n.py

```diff
@@ -19,12 +21,15 @@ def get_db_strings():
     elements = defaultdict(set)
     for entry in AppText.query:
         if entry.custom_text:
-            elements['"{}"'.format(entry.custom_text)].add("apptext: " + entry.name)
+            ct = re.sub('"', r'\\"', entry.custom_text)
```

Comment:
@ivan-c Wouldn't that be (slightly) less efficient though, cycling back through elements again?

Would it be better just to move the `re.sub` and insertion into `elements` into a separate method, just to avoid duplicating code and improve readability?